job_definitions/notify_suppliers_whether_application_made_for_framework.yml

```diff
@@ -43,5 +46,9 @@
             FLAGS="$FLAGS --dry-run"
           fi
 
+          if [ -n "$SUPPLIER_IDS"; then
+            $FLAGS="$FLAGS --supplier-ids='$SUPPLIER_IDS'"
```

Comment:
I think the script has `--supplier-id` singular?

Reply:
Yep!

Reply:
Fixed 😁